src/server.h

```diff
@@ -1633,6 +1633,10 @@ struct valkeyServer {
     list *replicas, *monitors;             /* List of replicas and MONITORs */
     client *current_client;                /* The client that triggered the command execution (External or AOF). */
     client *executing_client;              /* The client executing the current command (possibly script or module). */
+    in_addr_t host_machine_ip;             /*Listening ip for host machine network*/
+    in_addr_t host_machine_netmask;        /*Netmask for host_machine_ip*/
+    in_addr_t *trustedIPList;
+    unsigned int trustedIPCount;
```

Comment:
Please use snake casing and add comments
```suggestion
    in_addr_t *trusted_ip_list;
    unsigned int trusted_ip_count;
```